extension/app/src/components/conversation/AttachFragment.tsx

```diff
@@ -25,6 +25,21 @@ export const AttachFragment = ({
     return () => clearTimeout(timer);
   }, [attachPageBlinking]);
 
+  const uploadContentTab = async ({
+    includeContent,
+    includeCapture,
+  }: {
+    includeContent: boolean;
+    includeCapture: boolean;
+  }) => {
+    setIsSubmitting(true);
+    await fileUploaderService.uploadContentTab({
```

Comment:
the `fileUploaderService` already has a `isProcessingFiles` prop, can't we use it ? So we can keep isSubmitting and isProcessingFiles separated